Cargo.toml

```diff
@@ -16,18 +16,21 @@ lazy_static = "1.4.0"
 pin-project = { version = "0.4.6", optional = true }
 prometheus = { version = "0.7.0", optional = true }
 rand = { version = "0.7.2", optional = true }
+serde = { version = "1.0.104", features = ["derive"], optional = true }
 
 [dev-dependencies]
 hyper = "0.12.0"
 opentelemetry-jaeger = { path = "opentelemetry-jaeger", version = "0.1.0" }
 thrift = "0.13.0"
 futures = "0.3.1"
 tokio = { version = "0.2.10", features = ["full"] }
+bincode = "1.2.1"
```

Comment:
Should probably mark this as optional as well and include in the serialize feature

Reply:
@jtescher - Done. I had thought about this but it appeared to be the only way to resolve the issue: it is a test dependency (only) so naturally lives under `dev-dependencies`. But since it's only required for the `serialize` feature and since `dev-dependencies` cannot be optional, I assumed it would have to stay there. However, it seems that making it a dependency of the `serialize` feature as you suggest dtrt (`bincode` isn't linked into the resulting binaries regardless of whether `serialize` is enabled or not ;)